### ISSUE TITLE
Tertiaire link op highlight box geeft te weinig contrast

### ISSUE THREAD

**Issue by WiebeCnossen** [dso-toolkit/dso-toolkit#626]

**Comment by bramhoosemans** [comment]
Voorbeeld 'lees meer' in de toelichting.

**Comment by gofix** [comment]
@MichielDHVD jij zou hier een voorstel doen toch?

**Comment by MichielDHVD** [comment]
Tertiare knop in toelichtingen en highlight box moet zwart worden zoals de andere links ook zwart worden.  #191919 

![Toelichting met tertiare knop](https://user-images.githubusercontent.com/36912722/82891630-57f28b00-9f4e-11ea-81a4-04f67cb770f5.jpg)

@bramhoosemans  eens?

**Comment by bramhoosemans** [comment]
@MichielDHVD Ja, ik zie geen andere 'mooie' oplossing. Het lijkt me wel goed dat we in documentatie toevoegen dat er altijd een icoon bij geplaatst moet worden (anders lijkt het H5).

**Comment by bramhoosemans** [comment]
@MichielDHVD states toevoegen :)

**Comment by MichielDHVD** [comment]
![Toelichting Hover + Focus](https://user-images.githubusercontent.com/36912722/83145469-1f40e600-a0f5-11ea-9c8b-a1a9ec0ba066.jpg)

kleur voor hover en active states wordt #666666 komt door de wcag contrast checker heen met 4.55 dus kan net. Plus er komt een underline bij want die zit al op de tertiare button.

**Comment by PieterVerheij** [comment]
De styling van de primaire en secundaire button blijft hetzelfde, neem ik aan?

![image](https://user-images.githubusercontent.com/25430835/83241520-3c2ff480-a19b-11ea-8aac-13e10a3b8b25.png)

**Comment by MichielDHVD** [comment]
Ja deze blijft voorlopig hetzelfde

**Comment by PieterVerheij** [comment]
Zie https://www.dso-toolkit.nl/_626.Tertiaire-link-op-highlight-box-geeft-te-weinig-contrast/components/detail/buttons.html

**Comment by bramhoosemans** [comment]
Het icoon lijkt nog 'paars' te zijn, in ieder geval op hover.

**Comment by PieterVerheij** [comment]
Idd op hover, fixed! https://www.dso-toolkit.nl/_626.Tertiaire-link-op-highlight-box-geeft-te-weinig-contrast/components/detail/highlight-box.html

**Comment by bramhoosemans** [comment]
Top!
